src/components/QuranReader/ReadingView/WordByWordVerseAction/index.tsx

```diff
@@ -122,13 +122,13 @@ const WordByWordVerseAction: React.FC<Props> = ({
               const words = data.verses.map((verseItem) => getVerseWords(verseItem)).flat();
 
               return (
-                <>
+                <div data-testid="wbw-verse-modal-content">
```

Comment:
Introducing a `div` element here might change the UI. We can use `React.Fragment` to achieve the same thing